### ISSUE TITLE
Web map title or caption

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/Web-Map-Custom-Element#736]
### Discussed in https://github.com/Maps4HTML/Web-Map-Custom-Element/discussions/558

<div type='discussions-op-text'>

<sup>Originally posted by **prushforth** November  2, 2021</sup>
When we listen to what a screen reader says about a map, I hear the page title and then "Application", "Interactive map", "Region", but I don't hear what the title of the map is.  This seems to me to be a shortcoming, I think each map should have a title (optionally) that is announced, and potentially visible somewhere, based on author  discretion.  

WDYT? </div>

**Comment by prushforth** [comment]
We are thinking about creating a \<map-caption> child element for the \<mapml-viewer>.  The element's first job will be to generate an aria-label that is read by screen readers.  We can / may evolve the functionality to allow visual rendering of the caption, based on what you can do with \<figure> and \<figcaption>.  

We were unsure if \<figure> had the appropriate semantics to wrap a map.  If \<map-caption> turns out to be functionally identical to \<figcaption>, we can adopt the latter into our proposal.  But creating the \<map-caption> custom element will be a useful excercise, as well as useful for authors and especially users.
